### PR TITLE
added 5 of each and seperated, added icon

### PR DESCRIPTION
**Pull request by CodingSpecies**
I added the new icon, I changed to only add 5 as a preview, and added the rest to the seperate files. #56

### CODE REVIEW

**Review by AvidCoder101 — APPROVED** [review]
Thanks for the changes. LGTM

File: README.md [hunk 1]
```diff
@@ -147,36 +145,4 @@ Pull requests are welcome 👍. For major changes, please open an issue first to
 
 [MIT](https://choosealicense.com/licenses/mit/)
 
-
 ------------------------------------
-
-## Contributors ✨
```
Comment (MRslimcoder) [inline README.md:153]: This will remove add contributor bots bedge.

**Review by MRslimcoder — CHANGES_REQUESTED** [review]
Everything looks good just stash changes of all contributors bot don't remove them. 

Thank you!

File: README.md (hunk 1 above)
Comment (CodingSpecies) [reply README.md:153]: I don't know why that happened

**Comment by CodingSpecies** [comment]
Done

**Review by MRslimcoder — APPROVED** [review]
LGTM

**Pull request merged**